source/mutex/src/raw_impls.rs

```diff
@@ -269,3 +269,53 @@ pub mod lock_api_0_4 {
         }
     }
 }
+
+#[cfg(feature = "std")]
+pub mod std {
+    //! Std implementation of the [RawMutex] trait
+    //!
+    //! Currently based on [`parking_lot::RawMutex], but subject to change
+
+    use parking_lot::{lock_api::RawMutex as _, RawMutex as PLRawMutex};
+    use mutex_traits::{ConstInit, RawMutex};
+
+    /// Std implementation of the [RawMutex] trait
+    ///
+    /// Currently based on [`parking_lot::RawMutex], but subject to change
+    #[cfg_attr(feature = "fmt", derive(Debug))]
+    pub struct StdRawMutex {
+        inner: PLRawMutex,
+    }
+
+    impl ConstInit for StdRawMutex {
+        const INIT: Self = StdRawMutex { inner: PLRawMutex::INIT };
+    }
+
+    unsafe impl RawMutex for StdRawMutex {
+        type GuardMarker = ();
```

Comment:
@hawkw this is probably wrong but is there something you'd suggest that is more right here to signal that we're sync/send?

Reply:
I think this should be `<parking_lot::RawMutex as lock_api::RawMutex>::GuardMarker`...which I actually think is _not_ `Send`.

Reply:
Fixed!